create_graph.py

```diff
@@ -101,7 +104,7 @@ def autolabel(rects):
             color = graph_data["color"]
             std = graph_data["std"]
             bar = ax.bar(x_pos, avg, width=width, color=color)
-            ax.errorbar(x_pos, avg, yerr=std, fmt=",-r", ecolor="red")
+            ax.errorbar(x_pos, avg, yerr=std/2, fmt=",-r", ecolor="red")
```

Comment:
I don't think it's appropriate to halve the error bars. See for example https://stats.stackexchange.com/questions/353276/should-error-bars-be-halved.
```suggestion
            ax.errorbar(x_pos, avg, yerr=std, fmt=",-r", ecolor="red")
```